orchagent/fdborch.cpp

```diff
@@ -440,38 +446,68 @@ void FdbOrch::updateVlanMember(const VlanMemberUpdate& update)
 bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name, const string& type)
 {
     SWSS_LOG_ENTER();
-
+#if 0
     if (m_entries.count(entry) != 0) // we already have such entries
     {
         // FIXME: should we check that the entry are moving to another port?
         // FIXME: should we check that the entry are changing its type?
         SWSS_LOG_ERROR("FDB entry already exists. mac=%s bv_id=0x%lx", entry.mac.to_string().c_str(), entry.bv_id);
         return true;
     }
-
+#endif
     sai_fdb_entry_t fdb_entry;
 
     fdb_entry.switch_id = gSwitchId;
     memcpy(fdb_entry.mac_address, entry.mac.getMac(), sizeof(sai_mac_t));
     fdb_entry.bv_id = entry.bv_id;
 
     Port port;
-    /* Retry until port is created */
-    if (!m_portsOrch->getPort(port_name, port))
+    sai_object_id_t bridge_port_id = SAI_NULL_OBJECT_ID;
+    VxlanTunnelOrch* tunnel_orch = gDirectory.get<VxlanTunnelOrch*>();
+    VxlanTunnel *tunnel_obj = NULL;
+    if(strncmp(port_name.c_str(),"VTT",3) == 0)
```

Comment:
What is VTT? We would need to define this in a common place.

Reply:
VTT is the abbreviation of vxlan tunnel name, such as VTTNL0001. We will define it in common place.Thanks.

Reply:
we has fixed it.